opentreemap/treemap/units.py

```diff
@@ -31,6 +31,10 @@ def terminology(cls, instance=None):
                          .get(cls.__name__, {}))
         return terms
 
+    @classmethod
+    def display_name(cls, instance):
```

Comment:
the `display_name` method was phased on on several models in favor of using the terminology API.

https://github.com/steventlamb/otm-core/commit/286fe6c85e295607dc6acd360b407fe2ba420b7f#diff-eb9b8ea2fa5bef48515b23b723bbd6d4
https://github.com/steventlamb/otm-core/commit/d7a866b12e1dd9174dbc49607e0ace751e938783#diff-eb9b8ea2fa5bef48515b23b723bbd6d4
https://github.com/steventlamb/otm-core/commit/286fe6c85e295607dc6acd360b407fe2ba420b7f#diff-eb9b8ea2fa5bef48515b23b723bbd6d4